src/class_objectProcessor.py

```diff
@@ -539,7 +527,7 @@ def processmsg(self, data):
                 except:
                     apiNotifyPath = ''
                 if apiNotifyPath != '':
-                    call([apiNotifyPath, "newMessage"])
+                    call([apiNotifyPath, "newMessage"])  # nosec
```

Comment:
Ok but perphaps there could be a TODO for sanitisation? Not sure at the moment.

Reply:
Personally, I believe that any commit should be as good as we can make it; production quality. If we have TODOs and commented out code, this is a sign that we have lost our agile courage to change the code with confidence. In that vein, let's fully consider the consequences of this code and whether it should be #nosec'ed or not right now.

As far as I can see, apiNotifyPath is fetching a string from settings on the disk to use as the call() command, and the parameter to call() is a hard-coded string. If anyone had access to edit that setting to cause call() to do something bad then they have local access anyway and could just run arbitrary code if they wanted.

If anyone has any concerns that the call() command could in any way run code from the network then we should address that now rather than #nosec'ing it.